common/check_page_status/check_page_status.py

```diff
@@ -0,0 +1,44 @@
+from urllib.request import Request, urlopen
+import json
+import requests
+import argparse
+import logging
+
+
+def get_sitemap(url):
+    req = Request(url + 'sitemap/?format=json')
+    req.add_header('User-Agent', 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.106 Safari/537.36 OPR/38.0.2220.41')
+    req.add_header('Accept', 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8')
+    req.add_header('Accept-Charset', 'ISO-8859-1,utf-8;q=0.7,*;q=0.3')
+    req.add_header('Accept-Encoding', 'none')
+    req.add_header('Accept-Language', 'en-US,en;q=0.8')
+    content = json.loads(urlopen(req).read().decode('utf-8'))
+    return content
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Check status of all pages of a given IoGT site.')
```

Comment:
if easier could you convert this to a management command?

Reply:
How would I do that?